module-services/service-cellular/service-cellular/call/CellularCall.hpp

```diff
@@ -35,7 +35,9 @@ namespace CellularCall
     class Call
     {
         CalllogRecord call;
-        bool isActiveCall = false;
+        bool isActiveCall      = false;
+        bool wasRinging        = false;
```

Comment:
A set of flags had to be added to complete the current flow. It will probably be solved better with ongoing call related tasks.